packages/styles/src/tool-header.scss

```diff
@@ -244,21 +262,295 @@ $fd-tool-header-object-status-semantic-values: (
     }
   }
 
-  .#{$block}__input-group {
-    background: var(--sapShellColor);
-    border: 0.0625rem solid var(--sapShell_InteractiveBorderColor);
+  .#{$block}__search-field {
+    --fdSelect_Text_Content_Color: var(--fdShellbar_Select_Content_Color);
+    --fdInputGroup_Input_Border: var(--fdShellbar_Input_Border);
+    --fdInputGroup_Input_Color: var(--fdShellbar_Input_Color);
+    --fdInputGroup_Input_Placeholder_Color: var(--fdShellbar_Input_Placeholder_Color);
+    --fdInputGroup_Input_Placeholder_Style: var(--fdShellbar_Input_Placeholder_Style);
+    --fdInputGroup_Background: none;
+    --fdInputGroup_Background_Color: var(--fdShellbar_Input_Background);
```

Comment:
all the variables starting with `--fdShellbar....` should be duplicated for Tool Header, in my opinion. They should be moved to their own files and renamed to `--fdToolHeader....`
This will avoid the situation where the design for ShellBar changes but for Tool Header doesn't. By using the same CSS variables you are making Tool Header dependant on ShellBar, and they shouldn't be.